### PR TITLE
Update README for v1.0

### PR DESCRIPTION
**Pull request by frances-h**
View the new README here: https://github.com/sdv-dev/SDV/tree/readme-update-v1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,220 +58,119 @@ hierarchical generative modeling and recursive sampling techniques.
...
+    column_name='amenities_fee',
+    metadata=metadata
+)
+    
+fig.show()
 ```
 
-The output will be a dictionary with the same structure as the original `tables` dict,
-but filled with synthetic data instead of the real one.
+![Real vs. Synthetic Data](https://github.com/sdv-dev/SDV/blob/readme-update-v1.0/docs/images/Real-vs-Synthetic-Evaluation.png)
```
Comment (frances-h) [inline README.md:157]: Need to update link to point to master branch before merging

File: README.md [hunk 2]
```diff
@@ -57,220 +58,119 @@ hierarchical generative modeling and recursive sampling techniques.
...
+    dataset_name='fake_hotel_guests')
 ```
 
-# Quickstart
-
-**Are you using the new SDV 1.0 Beta?** Get started [with these new demos](https://docs.sdv.dev/sdv/demos)!
-
-In this short tutorial we will guide you through a series of steps that will help you
-getting started using **SDV**.
+![Single Table Metadata Example](https://github.com/sdv-dev/SDV/blob/readme-update-v1.0/docs/images/Single-Table-Metadata-Example.png)
```
Comment (frances-h) [inline README.md:89]: Need to update link to master branch before merging

**Comment by amontanez24** [comment]
This is a pre-existing issue but our logo is hard to read if the viewer is in dark mode 
Screen Shot 2023-03-23 at 10 15 19 PM

**Review by npatki — COMMENTED** [review]
LGTM

**Comment by npatki** [comment]
> This is a pre-existing issue but our logo is hard to read if the viewer is in dark mode Screen Shot 2023-03-23 at 10 15 19 PM

We can create non-PNG versions of this logo so it always has a white background. Does the same happen for the "DataCebo" logo at the bottom of the README? @amontanez24

**Comment by frances-h** [comment]
> > This is a pre-existing issue but our logo is hard to read if the viewer is in dark mode Screen Shot 2023-03-23 at 10 15 19 PM
> 
> We can create non-PNG versions of this logo so it always has a white background. Does the same happen for the "DataCebo" logo at the bottom of the README? @amontanez24

Yeah, similar issue. If we have dark-mode versions of the logos we could also dynamically swap between the light/dark mode versions based on the viewer's theme context. 

Screenshot 2023-03-24 at 10 29 52 AM

**Comment by npatki** [comment]
> Yeah, similar issue. If we have dark-mode versions of the logos we could also dynamically swap between the light/dark mode versions based on the viewer's theme context.

Good to know! We do for DataCebo but not yet for the logo at the top. I'll share them with you.

File: README.md [hunk 3]
```diff
@@ -291,7 +191,10 @@ If you use **SDV** for your research, please consider citing the following paper
 
 
 <div align="center">
-<a href="https://datacebo.com"><img align="center" width=40% src="https://github.com/sdv-dev/SDV/blob/master/docs/images/DataCebo.png"></img></a>
+  <a href="https://datacebo.com"><picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://github.com/sdv-dev/SDV/blob/readme-update-v1.0/docs/images/datacebo-logo-dark-mode.png">
+      <img align="center" width=40% src="https://github.com/sdv-dev/SDV/blob/readme-update-v1.0/docs/images/datacebo-logo.png"></img>
+  </picture></a>
```
Comment (frances-h) [inline README.md:197]: Need to update links to point to master before merging

**Review by amontanez24 — COMMENTED** [review]
:shipit:

**Review by amontanez24 — APPROVED** [review]

**Pull request merged**